.ci/docker/common/install_cuda.sh

```diff
@@ -3,7 +3,7 @@
 set -ex
 
 NCCL_VERSION=v2.21.5-1
-CUDNN_VERSION=9.1.0.70
```

Comment:
Reminder that we will not be using this variable but hard-code cudnn version to be 9.1.0.70 in install_118 step in this file.